.circleci/config.yml

```diff
@@ -36,7 +36,6 @@ jobs:
       # This may be unnecessary once rules_nodejs uses nodejs 8
       - run: bazel run @nodejs//:npm run postinstall
       - run: bazel build src/...
-      - run: bazel test src/...
```

Comment:
Comment out instead of removing?

Reply:
It will be a while before we can fix this and once we do fix it we won't just forget to turn it back on.